lib/util.dart

```diff
@@ -3,3 +3,6 @@ import 'dart:collection';
 String getSongTitle(LinkedHashMap song) {
   return (song['number'] != null ? song['number'] + ': ' : '') + song['title'];
 }
+
+// HACK - needs refactor
```

Comment:
Hmm?

Reply:
Nem akartam 4 szinten át passzolni a jelenlegi kiválasztott énekeskönyv "rész-mapjét", Dartban egyébként is safe és praktikus a global használat. Ettől még hack - a songsPage refaktorral meg fog szűnni a szükségessége, ezt jeleztem.

Reply:
Sőt bocsánat - a rész-map elérhető songspageből, de a dialognak pont a cross linking miatt a teljesre szüksége van. És: igazából csak az index számoláshoz, ha mondhatna közvetlen könyvet és "rendes énekszámot" a songsPagenek, nem is kéne neki a teljes json.

Reply:
Hehe… a jsonban rendes énekszám van – a trükk ugye az, hogy nem folyamatos a számozás, ezért kell map. Ami meg ugye másképp indexelődik, vagymi… ;)